pynxtools/dataconverter/readers/stm/stm_file_parser.py

```diff
@@ -0,0 +1,383 @@
+"""
+    A parse stm file in nexus generated definition template.
+"""
+
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+from typing import Any, Dict
+import logging
```

Comment:
This logger I remember @sherjeelshabih rightly so made once the comment in one the readers I wrote that we should use one of the official loggers so that nomad can pick up issues and problems in how far is this realized here? Maybe a topic for the future?

Reply:
I fully support your point of view. Indeed, I started to think about that but I saw that the repo needs to be restored again.